scripts/dummy_data.sql

```diff
@@ -14,9 +14,12 @@ INSERT INTO conf_admin.room(
     select 'Short Description', 'Full Description', conference.id
                from conf_admin.conference where conference.name = 'Sample Conference';
 
+insert into conf_admin.company
+(name, short_desc, full_desc, logo_path) values ('GFS', 'Gordon Food Service', 'Food distribution', 'http://logo_path/gfs.png');
+
 INSERT INTO conf_admin.presenter(
             user_id, short_bio, job_title, company_affiliation_id)
-    VALUES (null, 'short bio', 'sample title', null);
+    VALUES (null, 'short bio', 'sample title', 1);
```

Comment:
I am not going to implement this change.  please understand that this dummy_data.sql is only used for testing and nothing more.